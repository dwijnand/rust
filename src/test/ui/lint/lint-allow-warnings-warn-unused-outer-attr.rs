// Tests #[warn(unused_variables)] overrides and triggers a warning

// compile-pass

#[allow(warnings)]
#[warn(unused_variables)]
fn main() {
    let x = 3;
}
