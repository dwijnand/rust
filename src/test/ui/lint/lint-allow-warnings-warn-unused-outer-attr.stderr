warning: unused variable: `x`
  --> $DIR/lint-allow-warnings-warn-unused-outer-attr.rs:8:9
   |
LL |     let x = 3;
   |         ^ help: consider using `_x` instead
   |
note: lint level defined here
  --> $DIR/lint-allow-warnings-warn-unused-outer-attr.rs:6:8
   |
LL | #[warn(unused_variables)]
   |        ^^^^^^^^^^^^^^^^

